benchmarks/bench_rcv1_logreg_convergence.py

```diff
@@ -140,6 +140,15 @@ def plot_dloss(clfs):
         plt.xlabel("seconds")
         plt.ylabel("log(best - train_loss)")
 
+def get_max_squared_sum(X):
+    """
```

Comment:
PEP257: one-line summary belongs here: "Get the maximum row-wise sum of squares" or something.